mock_scripts

```diff
@@ -0,0 +1,8 @@
+#! /usr/bin/env bash
+
+set -ex
+
+mkdir -p internal/mock
+
+# Mock for survey package
+mockgen -destination internal/mock/prompt/survey.go -package mock -source prompt/survey.go
```

Comment:
Please use go-generate based mock generation.
It reduces maintenance overhead that comes with having separate script to generate mocks.

Reply:
I did not know this could be done. Will make the change.